api/tests/unit/domain/models/user_test.js

```diff
@@ -95,4 +95,20 @@ describe('Unit | Domain | Models | User', () => {
 
   });
 
+  describe('#email', function() {
+
+    it('should normalize email', () => {
```

Comment:
préciser le traittement par rapport à l'exemple donné

Reply:
Out of scope